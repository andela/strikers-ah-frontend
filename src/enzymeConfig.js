import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-localstorage-mock';

export default Enzyme.configure({
  adapter: new Adapter(),
});
