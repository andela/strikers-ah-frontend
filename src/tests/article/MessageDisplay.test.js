import React from 'react';
import { mount } from 'enzyme';
import '../../enzymeConfig';

import {
  MessageDisplay,
  mapStateToprops,
} from '../../components/article/MessageDisplay';

const articleUpdate = {
  message: 'fafasfsaf',
  error: 'fafafa',
};
const props = {
  sweetalert: jest.fn(),
  articleUpdate,
  article: {
    error: 'error',
    article: [],
  },
};

const wrapper = mount(<MessageDisplay {...props} />);
describe('test for `MessageDisplay` component', () => {
  test('should render the `MessageDisplay`', () => {
    expect(wrapper).toMatchSnapshot();
  });
  test('should test `UpdateMessageDisplay`', () => {
    const mockstate = {
      Article: {
        article: {},
      },
    };
    const state = mapStateToprops(mockstate);
    expect(state).toBeTruthy();
  });
});
