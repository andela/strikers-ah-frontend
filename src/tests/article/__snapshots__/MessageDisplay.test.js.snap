// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for \`MessageDisplay\` component should render the \`MessageDisplay\` 1`] = `ReactWrapper {}`;
